server/controllers/contest.js

```diff
@@ -85,4 +86,31 @@ exports.getContests = asyncHandler(async (req, res, next) => {
     } catch (error) {
     return res.status(500).send({ error });
     }
+})
+
+
+exports.createSubmission = asyncHandler(async (req, res, next) => {
+    const userID = req.user.id;
+    const contestID = req.params.id;
+    const { imageFiles } = req.body;
+    const active = true;
+    const submission = new Submission({
```

Comment:
Looks good! Maybe you would also want to check to see if a submission from the current user already exists? If yes, update the existing submission. If no, then maybe create a new one?